### PR TITLE
lock csi and plugin watcher GA feature gates

### PR DESCRIPTION
**Pull request by msau42**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
GA features cannot be disabled

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
The CSIPersistentVolume and KubeletPluginWatcher feature gates cannot be disabled, and will be removed in Kubernetes v1.16
```

### CODE REVIEW

File: test/integration/auth/node_test.go [hunk 1]
```diff
@@ -602,12 +599,7 @@ func TestNodeAuthorizer(t *testing.T) {
 	expectAllowed(t, updatePVCCapacity(node2Client))
 	expectForbidden(t, updatePVCPhase(node2Client))
 
-	// Disabled CSIPersistentVolume feature
-	defer utilfeaturetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.CSIPersistentVolume, false)()
-	expectForbidden(t, getVolumeAttachment(node1ClientExternal))
```
Comment (msau42) [inline test/integration/auth/node_test.go]: I removed the node authorizer tests with the csi feature disabled. cc @liggitt

**Comment by liggitt** [comment]
/lgtm

add a release note that these two GA feature gates cannot be disabled, and the feature gates will be removed in 1.16

**Comment by msau42** [comment]
Rebased

**Comment by msau42** [comment]
Rebased again

**Pull request merged**